transform/instance.go

```diff
@@ -317,7 +317,7 @@ func (ot *objectTransformer) selectChild(key string) instanceTransformer { retur
 
 // transform retrieves the value for this object by building the value for the base object and then adding in any
```

Comment:
You changed the function name but need to fix up the godocs to match, this is true for all xml and updated transform function as well.